import-service/serverless.yml

```diff
@@ -0,0 +1,59 @@
+service: import-service
+
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+  lambdaHashingVersion: 20201221
+  stage: dev
+  region: eu-west-1
+
+  iam:
+    role: 
+      statements:
+        - Effect: "Allow"        
+          Action: "s3:*"        
+          Resource: 
+            - "arn:aws:s3:::rs-school-import-service-bucket"
+            - "arn:aws:s3:::rs-school-import-service-bucket/*"
+        - Effect: "Allow"        
+          Action: "s3:ListBucket"        
+          Resource: 
+            - "arn:aws:s3:::rs-school-import-service-bucket"
+
```

Comment:
I recommend using the Resources section in the config file to create the bucket and to provide its' CORS.

Reply:
I followed the task description. In Prerequisites section we have a point Sign in into AWS Console and create a new S3 bucket with a folder, called uploaded. That's why I created s3 manually by AWS Console.